contracts/RelayHub.sol

```diff
@@ -122,8 +122,11 @@ contract RelayHub is IRelayHub {
     private
     view
     returns (IPaymaster.GasLimits memory gasLimits, uint256 maxPossibleGas) {
-        gasLimits =
-            IPaymaster(relayRequest.relayData.paymaster).getGasLimits();
+
+        (bool success, bytes memory ret) = relayRequest.relayData.paymaster.staticcall{gas:50000}(abi.encodeWithSelector(IPaymaster.getGasLimits.selector));
```

Comment:
If I understand solidity correctly, it will compile the line with a call to view function as a staticcall, won't it?
If so, is there a benefit in going to `staticcall{gas:50000}` over `.getGasLimits{gas:50000}()` ? Just asking.

Reply:
nope... I forgot (or completely didn't know) you can put gas limit on any method call...